README.md

```diff
@@ -61,6 +61,7 @@ CMD [ "/bin/bash", "/usr/src/mystartscript.sh" ]
 
 ### Environment variables
 
-| Environment variable | Description                               | Default | Options |
-| -------------------- | ----------------------------------------- | ------- | ------- |
-| `DBUS_PORT`          | The port on which the bus listens         | 55884   | -       |
+| Environment variable | Description                               | Default      | Options |
+| -------------------- | ----------------------------------------- | -------      | ------- |
+| `DBUS_PORT`          | The port on which the bus listens         | 55884        | -       |
+| `DBUS_CONFIG`        | The config file to use for the bus        | session.conf | -       |
```

Comment:
Let's add relative to /`usr/src/app`? Some other users might want to override the configs and they can put it in that path